### PR TITLE
chore: Add specific build for public page

### PR DESCRIPTION
**Pull request by cballevre**
```
### 🔧 Tech

* Add specific build for public page
```

**Related PRs :**
- https://github.com/cozy/create-cozy-app/pull/1492

### CODE REVIEW

File: src/targets/public/index.ejs [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      href="./favicon-16x16.png"
+      sizes="16x16"
+    />
+    <link rel="manifest" href="/manifest.json" crossorigin="use-credentials" />
+    <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#297EF2" />
+    <% if (__TARGET__ !== 'mobile') { %>
```
Comment (Crash--) [inline src/targets/public/index.ejs:21]: You can remove it

File: src/targets/public/index.jsx [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+const init = () => {
+  const { appLocale, client } = memoizedInit()
+  renderApp(appLocale, client)
+}
+document.addEventListener('DOMContentLoaded', init)
+
+if (module.hot) {
+  memoizedInit()
+  module.hot.accept()
+}
```
Comment (Crash--) [inline src/targets/public/index.jsx:130]: Why do you have copy / pasted this thing, can't we use the other index.jsx?

Comment (cballevre) [reply src/targets/public/index.jsx:130]: @Crash-- to get an entrypoint so webpack generate an bundle for public which wasn't the case before. I agree with you that we can remove `module.hot` but I will check if removing hot reload doesn't have any impact in watch mode on public page development.

**Comment by cballevre** [comment]
@Crash-- can I have new review on this PR ? pls

**Review by Crash-- — APPROVED** [review]

**Pull request merged**